pkg/gen/client/config/clientset/versioned/typed/config/v1alpha1/meshconfig.go

```diff
@@ -60,6 +63,57 @@ func newMeshConfigs(c *ConfigV1alpha1Client, namespace string) *meshConfigs {
 	}
 }
 
+func (c *meshConfigs) GetDefaultConfig(resourceVersion string) (defaultConfig *v1alpha1.MeshConfig) {
```

Comment:
Let please create this in the common.go of the e2e, since it will be used only for the e2e tests and has hard coded values

Reply:
+1

This should not be updating the API.

Reply:
@shashankram @snehachhabria instead of creating a new meshconfig object, I will just override the fields I need to in the existing meshconfig. Otherwise, I am seeing that the envoy / init images that initially point to another registry (ex MCR) are being overridden and pointing to the osm dockerhub images instead every time the default is applied. Also, we don't necessarily want to hardcode values like EnablePermissiveTrafficPolicy in the tests,